files/en-us/web/api/gravitysensor/gravitysensor/index.html

```diff
@@ -0,0 +1,80 @@
+---
+title: GravitySensor.GravitySensor()
+slug: Web/API/GravitySensor/GravitySensor
+tags:
+- API
+- Accelerometer
+- Constructor
+- Generic Sensor API
+- GravitySensor
+- Reference
+- Sensor
+- Sensor APIs
+- Sensors
+---
+<div>{{APIRef("Generic Sensor API")}}</div>
+
+<p><span class="seoSummary">The <strong><code>GravitySensor</code></strong>
+    constructor creates a new {{domxref("GravitySensor")}} object which
+    provides on each reading the gravity applied to the device along all three axes.</span></p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre
+  class="brush: js">let <var>gravitySensor</var> = new GravitySensor([<em>options</em>])</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+  <dt><em>options</em> {{optional_inline}}</dt>
+  <dd>Options are as follows:
+    <ul>
+      <li><code>frequency</code>: The desired number of times per second a sample should
+        be taken, meaning the number of times per second that
+        {{domxref('sensor.onreading')}} will be called. A whole number or decimal may be
+        used, the latter for frequencies less than a second. The actual reading frequency
+        depends on device hardware and consequently may be less than requested. The default
+        frequency is the one defined by the underlying platform</li>
+      <li><code>referenceFrame</code>: The local coordinate system represents 
+        the reference frame. It can be either  <code>'device'</code> or
+        <code>'screen'</code>. The default is <code>'device'</code>.</li>
+    </ul>
+  </dd>
+</dl>
+
+<h3 id="Exceptions">Exceptions</h3>
+
+<dl>
+  <dt>SecurityError</dt>
+  <dd>If a feature policy blocks use of a feature, it is because your code is inconsistent
```

Comment:
```suggestion
  <dd>Use of this feature was blocked by a feature policy. If a feature policy blocks use of a feature, it is because your code is inconsistent
```

It would be good to have an introductory sentence, like the one in this suggestion, to link this exception to the feature policy bit.

Reply:
correct.